### PR TITLE
add TiKV test

### PR DESCRIPTION
**Pull request by zhijian-pro**
add TiKV test

### CODE REVIEW

**Comment by zhijian-pro** [comment]
It takes about ten minutes, I think it’s acceptable

File: pkg/meta/tkv_test.go [hunk 1]
```diff
@@ -72,3 +72,22 @@ func TestMemKV(t *testing.T) {
 		t.Fatalf("counter should be 0, but got %d", count)
 	}
 }
+
+func TestTiKV(t *testing.T) {
```
Comment (SandyXSD) [inline pkg/meta/tkv_test.go:76]: TestTiKVClient, and put it after MemKVClient

File: pkg/meta/tkv_test.go [hunk 2]
```diff
@@ -72,3 +72,22 @@ func TestMemKV(t *testing.T) {
...
+	m, err := newKVMeta("tikv", "127.0.0.1:2379/jfs", &Config{MaxDeletes: 1})
+	if err != nil {
+		t.Fatalf("create meta: %s", err)
+	}
+
+	testTruncateAndDelete(t, m)
```
Comment (SandyXSD) [inline pkg/meta/tkv_test.go:82]: update all the tests, same as MemKVClient

**Review by SandyXSD — CHANGES_REQUESTED** [review]

File: pkg/meta/tkv_test.go [hunk 3]
```diff
@@ -46,6 +46,30 @@ func TestMemKVClient(t *testing.T) {
 	testReadOnly(t, m)
 }
 
+func TestTiKVClient(t *testing.T) {
+	m, err := newKVMeta("tikv", "127.0.0.1:2379/jfs", &Config{MaxDeletes: 1})
+	if err != nil {
```
Comment (SandyXSD) [inline pkg/meta/tkv_test.go:51]: || m.Name() != "tikv"

**Review by SandyXSD — APPROVED** [review]

**Pull request merged**